src/components/tree-detail/hooks/use-tree-age-classification.tsx

```diff
@@ -30,7 +30,9 @@ export function useTreeAgeClassification(
 		if (!treeData) {
 			return undefined;
 		}
-		return treeData.pflanzjahr === 0
+		return treeData.pflanzjahr === 0 ||
```

Comment:
This looks pretty long to parse. I tested it A little bit shorter. But not a blocker:
index.mjs
```js
export function checkPflanzjahr({ pflanzjahr }) {
  return Number(pflanzjahr) ? new Date().getFullYear() - pflanzjahr : undefined;
}

``` 

The tests:

index.test.mjs

```js
import assert from "node:assert/strict";
import test from "node:test";

import { checkPflanzjahr } from "./index.mjs";

test("test undefined", () => {
  const expected = undefined;
  const actual = checkPflanzjahr({});
  assert.strictEqual(expected, actual);
});

test("test null", () => {
  const expected = undefined;
  const actual = checkPflanzjahr({ pflanzjahr: null });
  assert.strictEqual(expected, actual);
});

test("test 0", () => {
  const expected = undefined;
  const actual = checkPflanzjahr({ pflanzjahr: 0 });
  assert.strictEqual(expected, actual);
});

test("test 2023", () => {
  const expected = 1;
  const actual = checkPflanzjahr({ pflanzjahr: 2023 });
  assert.strictEqual(expected, actual);
});

test("test 2000", () => {
  const expected = 24;
  const actual = checkPflanzjahr({ pflanzjahr: 2000 });
  console.log(actual);
  assert.strictEqual(expected, actual);
});

test("test '2000' string", () => {
  const expected = 24;
  const actual = checkPflanzjahr({ pflanzjahr: "2000" });
  console.log(actual);
  assert.strictEqual(expected, actual);
});

test("test 'abc' string", () => {
  const expected = undefined;
  const actual = checkPflanzjahr({ pflanzjahr: "abc" });
  console.log(actual);
  assert.strictEqual(expected, actual);
});

```

Reply:
Thanks, we can use it in a follow up PR.